lib/ui/myplot/myplot_viewmodel.dart

```diff
@@ -26,7 +29,26 @@ class MyPlotViewModel {
         selectedCrop: store.state.myPlotState.selectedCrop,
         goToDetail: (CropEntity crop) => store.dispatch(GoToCropDetailAction(crop)),
         fetchCrops: () =>
-            store.dispatch(new FetchCropListAction())
+            store.dispatch(FetchCropListAction())
     );
   }
+
+  List<CropDetailProperty> getCropDetailProperties(CropEntity cropEntity) {
```

Comment:
Are these the only properties displayed in the crop details page? 

In the CMS there are more properties such as: 

crops in rotation
complexity
crop type
soil type (a list)
water requirement
setup cost
profitability
companion plants (list)
non-companion plants (list)

Reply:
This, indeed, are the ones we are showing.
I can add those now then :)